### ISSUE TITLE
create-labels should use the same colors

### ISSUE THREAD

**Issue by rbellamy** [intuit/auto#1239]
**Is your feature request related to a problem? Please describe.**

I just ran `create-labels` for the second time, and the two projects have different label colors.

**Describe the solution you'd like**

For consistency, I'd like the labels to be the same color each time I initialize them.

**Describe alternatives you've considered**

Alternatively, support exporting the colors of an already-configured repository for easy inclusion in `.autorc`.

**Follow-up question**

If I change my `.autorc` and reset the colors, and then re-run `create-labels`, will the new colors be applied?

**Comment by rbellamy** [comment]
Answering my own question about re-running `create-labels`.

Went here: https://gist.github.com/MoOx/93c2853fee760f42d97f#gistcomment-3069259

Exported labels, added the JSON to my `.autorc` file, re-ran `create-labels` - colors are still different.

**Comment by rbellamy** [comment]
To get the colors correct:
1. Delete all previous labels
2. Re-run `create-labels` with the desired colors in `.autorc`.

**Comment by hipstersmoothie** [comment]
Where we update labels: https://github.com/intuit/auto/blob/master/packages/core/src/release.ts#L454

The actual update function 
https://github.com/intuit/auto/blob/master/packages/core/src/git.ts#L548

It should update the labels if it can I can look into what's happening later

> For consistency, I'd like the labels to be the same color each time I initialize them.

You can do this with a shared `.autorc` already.

```json
{
  "labels": [
    {
      "name": "major",
      "color": "blue"
    },
   // other labels
  ]
}
```

**Comment by rbellamy** [comment]
I've already done the work to get a shared `.autorc` and use it... my reason for requesting the feature is that the current functionality of `create-labels` isn't consistent. It doesn't require an `.autorc` file in order to set labels, and therefore shouldn't require it to set those labels colors to default values.

Another way of saying this is: if `create-labels` is intelligent enough to have predefined label names and descriptions, it should also have predefined colors.

**Comment by hipstersmoothie** [comment]
Ah yeah. we could probably ship with some defaults and it would avoid this issue

**Comment by hipstersmoothie** [comment]
my preferred colors for the semver labels (matches auto's colors):

- patch - #870048
- minor - #F1A60E
- major - #C5000B

still needs defaults 

- skip-release - orange
- release - green
- internal - dark grey
- documentation - light grey

**Comment by gaurivn** [comment]
I would like to work on this issue.

**Comment by hipstersmoothie** [comment]
Go for it!

**Comment by gaurivn** [comment]
Thank you

**Comment by rbellamy** [comment]
Also, for completeness, here's the default labels created for all new repositories: https://help.github.com/en/github/managing-your-work-on-github/about-labels#using-default-labels

**Comment by adierkens** [comment]
:rocket: Issue was released in [`v9.40.5`](https://github.com/intuit/auto/releases/tag/v9.40.5) :rocket:
